packages/docs/src/pages/components/progress/examples/ExValues.vue

```diff
@@ -5,7 +5,7 @@
             <b-field label="Type and Size">
                 <b-field grouped>
                     <b-select v-model="type" placeholder="Type">
-                        <option value="null">Default</option>
+                        <option value="">Default</option>
```

Comment:
@wesdevpro Good catch! The original code did not make sense, which specified `"null"` as a string.

Reply:
If you would take the path I suggested, you should replace this with `<option :value="undefined">Default</option>`.